docs/notebooks/failure_ego.pct.py

```diff
@@ -17,27 +17,9 @@
 # # EGO with a failure region
 
 # %%
-from dataclasses import astuple
-
-import gpflow
-from gpflow import set_trainable
-import matplotlib.pyplot as plt
 import numpy as np
 import tensorflow as tf
-from typing import Dict
-import trieste
-from trieste.utils import objectives
-from trieste.models.model_interfaces import VariationalGaussianProcess
-from trieste.models import ModelSpec
-from gpflow.models import VGP
-from gpflow.optimizers import NaturalGradient
 
-from util.plotting_plotly import plot_function_plotly, plot_gp_plotly, add_bo_points_plotly
-from util.plotting import plot_gp_2d, plot_function_2d, plot_bo_points
-
-# %%
-gpflow.config.set_default_jitter(1e-5)
```

Comment:
this particular line change is outside the scope of this PR. I couldn't resist, cos it makes the intro to all the notebooks the same